.gitignore

```diff
@@ -2,3 +2,6 @@ target
 Dist
 .idea
 learnositysdk.iml
+.DS_Store
+*/.DS_Store
+docs/quickstart/assessment/webapps/
```

Comment:
Why is this added to git ignore "docs/quickstart/assessment/webapps/"
add a new line in the end.

Reply:
that is for compiled folder